docs/user-guide/cli.md

```diff
@@ -214,6 +214,15 @@ Print a configuration used for the given input file:
 stylelint test.css --print-config
 ```
 
+
+### Example K - apply a custom formatter through a bash pipe
+
+Print a configuration used for the given input file:
+
+```shell
+stylelint -f json "*.css" 2>&1 | my-program-that-reads-JSON --option
+```
```

Comment:
[suggest] I have several suggestions below, but any thoughts?

- Remove a redundant empty line above the heading
- Clarify the heading words
- Correct the description
- Use a real use case (`jq`) in the example

````suggestion
### Example K - piping a report to another command

Use a report as input for another command through piping:

```shell
stylelint -f json "*.css" 2>&1 | jq .
```
````

Reply:
You are absolutely right, sorry about that. I have added an example that just counts all the individual warnings.